bundle/Templating/Twig/Extension/ContentViewRuntime.php

```diff
@@ -0,0 +1,212 @@
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolverInterface;
+use Symfony\Component\HttpKernel\Controller\ControllerReference;
+use Symfony\Component\HttpKernel\Controller\ControllerResolverInterface;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * Twig extension runtime for content rendering (view).
+ */
+class ContentViewRuntime
+{
+    /**
+     * @var \Symfony\Component\HttpFoundation\RequestStack
+     */
+    private $requestStack;
+
+    /**
+     * @var \Symfony\Component\HttpKernel\Controller\ControllerResolverInterface
+     */
+    private $controllerResolver;
+
+    /**
+     * @var \Symfony\Component\HttpKernel\Controller\ArgumentResolverInterface
+     */
+    private $argumentResolver;
+
+    /**
+     * @var \Netgen\Bundle\EzPlatformSiteApiBundle\View\Builder\ContentViewBuilder
+     */
+    private $viewBuilder;
+
+    /**
+     * @var \eZ\Publish\Core\MVC\Symfony\View\Renderer
+     */
+    private $viewRenderer;
+
+    /**
+     * @var \Psr\Log\LoggerInterface|\Psr\Log\NullLogger|null
+     */
+    private $logger;
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\RequestStack $requestStack
+     * @param \Symfony\Component\HttpKernel\Controller\ControllerResolverInterface $controllerResolver
+     * @param \Symfony\Component\HttpKernel\Controller\ArgumentResolverInterface $argumentResolver
+     * @param \Netgen\Bundle\EzPlatformSiteApiBundle\View\Builder\ContentViewBuilder $viewBuilder
+     * @param \eZ\Publish\Core\MVC\Symfony\View\Renderer $viewRenderer
+     * @param \Psr\Log\LoggerInterface|null $logger
+     */
+    public function __construct(
+        RequestStack $requestStack,
+        ControllerResolverInterface $controllerResolver,
+        ArgumentResolverInterface $argumentResolver,
+        ContentViewBuilder $viewBuilder,
+        Renderer $viewRenderer,
+        ?LoggerInterface $logger = null
+    ) {
+        $this->requestStack = $requestStack;
+        $this->controllerResolver = $controllerResolver;
+        $this->argumentResolver = $argumentResolver;
+        $this->viewBuilder = $viewBuilder;
+        $this->viewRenderer = $viewRenderer;
+        $this->logger = $logger ?? new NullLogger();
+    }
+
+    /**
+     * Renders the HTML for a given $content.
+     *
+     * @param string $viewType
+     * @param \eZ\Publish\API\Repository\Values\ValueObject $contentOrLocation
+     * @param array $parameters
+     *
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentType
+     * @throws \eZ\Publish\Core\Base\Exceptions\UnauthorizedException
+     *
+     * @return string The HTML markup
+     */
+    public function renderContentView(
+        string $viewType,
```

Comment:
How about placing content/location first? We are rendering a content in a specific view, not the other way around.

Reply:
Yes indeed, will change 👍

Reply:
Updated in https://github.com/netgen/ezplatform-site-api/pull/117/commits/6bb212e8833c915a0a0085a238d0c79eab6be04d